docs/sections/userQuickStart.md

```diff
@@ -0,0 +1,131 @@
+<link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# User Quick Start
+
+## Requirement
+We expect users to have basic knowledge of the following:
+- [x] Markdown
+- [x] HTML & CSS
+- [x] npm (Node.js)
+- [x] Command-line Environment
+
+## Installation

```

Comment:
Specifically mention how to install node, assuming the user doesn't know about node.